src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -407,7 +458,10 @@ public void CallSubProcess(Action<Message> handleMessage, IRequest request, ISon
             }
 
             public void WaitUntilSubProcessCalledByAnalyzer()
-                => callbackFromCLangReceived.WaitOne(20000);
+            {
+                var result = callbackFromCLangReceived.WaitOne(20000);
+                result.Should().BeTrue("Timed out waiting for subprocess to be called");
```

Comment:
Previously, the test would still have passed if the callback was never received.